website/docs/r/resource_management_private_link.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "Management"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_resource_management_private_link"
+description: |-
+  Manages a Resource Management Private Link.
+---
+
+# azurerm_resource_management_private_link
+
+Manages a Resource Management Private Link.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+resource "azurerm_resource_management_private_link" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of this Resource Management Private Link. Changing this forces a new Resource Management Private Link to be created.
+
+* `resource_group_name` - (Required) Specifies the name of the Resource Group within which this Resource Management Private Link should exist. Changing this forces a new Resource Management Private Link to be created.
+ 
+* `location` - (Required) The Azure Region where the Resource Management Private Link should exist. Changing this forces a new Resource Management Private Link to be created.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the Resource Management Private Link.
+
+* `private_endpoint_connections` - A list of private endpoint connections.
```

Comment:
(as above) we likely don't want to expose this field:

```suggestion
```

Reply:
fixed.